mvpa2/support/afni/lib_prep_afni_surf.py

```diff
@@ -286,19 +311,75 @@ def yesno2bool(d, k): # dict, key
             c['outvol_ext'] = '+' + c['outvol_view']
             c['outvol_fullext'] = c['outvol_ext'] + '.HEAD'
 
-
     return c
 
+
+
+def _ensure_expvol_is_plump(expvol):
```

Comment:
to test this function you would need just a single expvol.  Is it possible to generate it or just ship some small one with necessary features within `mvpa2/data`?